care/emr/api/viewsets/inventory/inventory_item.py

```
@@ -32,7 +32,7 @@ class InventoryItemViewSet(EMRRetrieveMixin, EMRListMixin, EMRBaseViewSet):
     pydantic_retrieve_model = InventoryItemRetrieveSpec
     filterset_class = InventoryItemFilters
     filter_backends = [filters.DjangoFilterBackend, OrderingFilter]
-    ordering_fields = ["created_date", "modified_date"]
+    ordering_fields = ["created_date", "modified_date", "net_content"]
```

Ruff (0.14.4)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)